src/v0/destinations/braze/transform.js

```diff
@@ -154,7 +154,7 @@ function getUserAttributesObject(message, mappingJson, destination) {
       if (destKey === 'gender') {
         value = formatGender(value);
       } else if (destKey === 'email' && value !== null) {
-        value = value?.toLowerCase();
+        value = value?.toString()?.toLowerCase();
```

Comment:
```suggestion
        value = value?.toString().toLowerCase();
```
Also can we consider a case where user provides an object for this email .
We have seen cases like this before resulting into bugsnag issues.
Also shouldn't email be always string so can't provide a typeof check rather than converting it into string which ofcourse won't be email

Reply:
we can neglect case where email sent is in object, customer should adhere to contract of RS and Rudderstack.